webapp/src/ts/services/enketo.service.ts

```diff
@@ -288,44 +289,66 @@ export class EnketoService {
       });
   }
 
-  private renderFromXmls(doc, wrapper, instanceData) {
+  private renderFromXmls(xmlFormContext: XmlFormContext) {
+    const { wrapper } = xmlFormContext;
```

Comment:
I wasn't sure how to approach this one, I figured it would give more context to these properties if we would use them from the object.  
In hindsight it's even more confusing to only destructure part of the properties. Pushing a change to destructure everytthing here and in the comment below.